example/webpack4-babel/package.json

```diff
@@ -12,7 +12,7 @@
     "jsdom": "^11.7.0",
     "karma-jasmine": "^1.1.1",
     "karma-jsdom-launcher": "^6.1.3",
-    "karma-webpack": "^3.0.0",
+    "karma-webpack": "^4.0.0",
```

Comment:
technically, karma-webpack 3 isn't meant to be used with webpack 4 (https://github.com/webpack-contrib/karma-webpack/commit/c1c0ee245afbc87e1fd8d2dcaa393946f0f32abf) so I bumped it for the webpack 4 example